project/DistributionPackage.scala

```diff
@@ -276,7 +276,7 @@ object DistributionPackage {
             "JAVA_OPTS" -> "-Dorg.jline.terminal.dumb=true"
           ).run
           // Poor man's solution to stuck index generation
-          val GENERATING_INDEX_TIMEOUT = 60 * 2 // 2 minutes
+          val GENERATING_INDEX_TIMEOUT = 60 * 4 // 2 minutes
```

Comment:
Forgot to cleanup or
```suggestion
          val GENERATING_INDEX_TIMEOUT = 60 * 4 // 4 minutes
```

Reply:
in ideal world
```scala
scala> {
     | import scala.concurrent.duration._
     | 4.minutes.toSeconds
     | }
val res4: Long = 240
```

Reply:
Good catch, I will add it to next PR. Didn't want to trigger yet another build, it would take ages and this needed to get it to nightly.